apps/builder/app/builder/features/workspace/canvas-tools/outline/outline.tsx

```diff
@@ -1,17 +1,50 @@
 import { useMemo } from "react";
-import { styled, type Rect } from "@webstudio-is/design-system";
+import { css, keyframes, type Rect } from "@webstudio-is/design-system";
 import { theme } from "@webstudio-is/design-system";
+import { cssVars } from "@webstudio-is/css-vars";
 
-const OutlineContainer = styled("div", {
+const angleVar = cssVars.define("angle");
+
+const propertyStyle = (
+  <style>{`
+    @property ${angleVar} {
```

Comment:
Keep in mind it's quite new feature for non-chromium browsers
https://caniuse.com/?search=%40property

Reply:
adding comment